### ISSUE TITLE
`{prefix}:<NA>` making it into output SSSOM 

### ISSUE THREAD

**Issue by kevinschaper** [monarch-initiative/monarch-gene-mapping#31]
It looks like something is up with the join to populate mappings and we're creating these weird mappings when an ID doesn't exist: 

example:
```
HGNC:45103	skos:exactMatch	OMIM:<NA>	semapv:UnspecifiedMatching
HGNC:34495	skos:exactMatch	OMIM:<NA>	semapv:UnspecifiedMatching
HGNC:21224	skos:exactMatch	OMIM:<NA>	semapv:UnspecifiedMatching
```

Unfortunately, there are 52k of them, spread out like this:

```
26730 HGNC -> OMIM
23417 HGNC -> UniProtKB
2692 HGNC -> ENSEMBL
  46 HGNC -> NCBIGene
```

**Comment by kevinschaper** [comment]
For our current stack, this isn't actually affecting anything since we look up IDs in the object column, and wouldn't look up `<NA>` (hopefully).

**Comment by glass-ships** [comment]
resolved in 0.1.5
